lib/rubocop/cop/rspec/expectation_target_method.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # Checks ExpectationTarget method.
+      #
+      # @example
+      #   # bad
+      #   expect(something).kind_of? Foo
+      #   is_expected == 42
+      #
+      #   # good
+      #   expect(something).to be_a Foo
+      #   is_expected.to eq 42
+      #
+      class ExpectationTargetMethod < Base
+        MSG = 'Do not use any method other than `.to` or `.not_to`.'
```

Comment:
I would suggest we rephrase the message to use a positive “use…” instead of the more negative “do not use…”
```suggestion
        MSG = 'Use `.to`, `.not_to` or `.to_not` to make an assertion.'
```
or "…to set an expectation." depending on how strict we follow RSpec terminology 😅

Reply:
Thank you for the suggestion. I thought "...to set an expectation" would be better, so I will modify the message to that.